### PR TITLE
Docs: Fix SQL formatting in Flink docs

### PR DESCRIPTION
**Pull request by TaoZex**
1.Fix URL link in flink-getting-started.md
Before：

After：
9ad0fc83646b9595fcec533150e740c

2.Fix docs format in flink-ddl.md
Before：
d0ebe493d93bbf01d345eca3d3b15b5
After：
163ad880a116c75d7f2c13cfd57bbf5

### CODE REVIEW

**Comment by TaoZex** [comment]
@hililiwei  @stevenzwu @Fokko @nastra @rdblue Hello, I am a contributor to iceberg for the first time. Could you please help me review it? Thank you.

File: docs/flink-getting-started.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ export HADOOP_CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath`
 ./bin/start-cluster.sh
 ```
 
-Start the Flink SQL client. There is a separate `flink-runtime` module in the Iceberg project to generate a bundled jar, which could be loaded by Flink SQL client directly. To build the `flink-runtime` bundled jar manually, build the `iceberg` project, and it will generate the jar under `<iceberg-root-dir>/flink-runtime/build/libs`. Or download the `flink-runtime` jar from the [Apache repository](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-flink-runtime-1.16/{{% icebergVersion %}}/).
+Start the Flink SQL client. There is a separate `flink-runtime` module in the Iceberg project to generate a bundled jar, which could be loaded by Flink SQL client directly. To build the `flink-runtime` bundled jar manually, build the `iceberg` project, and it will generate the jar under `<iceberg-root-dir>/flink-runtime/build/libs`. Or download the `flink-runtime` jar from the [Apache repository] (https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-flink-runtime-1.16/{{%icebergVersion%}}/).
```
Comment (nastra) [inline docs/flink-getting-started.md:80]: The link in https://iceberg.apache.org/docs/latest/flink/ works fine for me. Note that the markdown files are being served by [Hugo](https://gohugo.io/) so the original link here was correct

File: docs/flink-getting-started.md [hunk 2]
```diff
@@ -399,4 +399,4 @@ There are some features that are do not yet supported in the current Flink Icebe
 * Don't support creating iceberg table with computed column.
 * Don't support creating iceberg table with watermark.
 * Don't support adding columns, removing columns, renaming columns, changing columns. [FLINK-19062](https://issues.apache.org/jira/browse/FLINK-19062) is tracking this.
-* 
\ No newline at end of file
+*
```
Comment (nastra) [inline docs/flink-getting-started.md:402]: nit: probably better to revert changes to this file

**Review by nastra — APPROVED** [review]
LGTM once we revert changes to the getting started page

**Comment by nastra** [comment]
thanks @TaoZex for your contribution and welcome to the Iceberg community

**Pull request merged**